### PR TITLE
fixed comment

### PR DESCRIPTION
**Pull request by ervand96**

### CODE REVIEW

File: src/app/create-card/create-card.component.ts [hunk 1]
```diff
@@ -1,52 +1,50 @@
-import { Component, OnInit } from '@angular/core';
+import { Component } from '@angular/core';
 
 @Component({
   selector: 'app-create-card',
   templateUrl: './create-card.component.html',
   styleUrls: ['./create-card.component.css']
 })
-export class CreateCardComponent implements OnInit {
+
```
Comment (Shitikyan) [inline src/app/create-card/create-card.component.ts]: extra line

File: src/app/create-card/create-card.component.ts [hunk 2]
```diff
@@ -1,52 +1,50 @@
...
   createCard = 'Create New Card';
   titleButtonClick = "Click";
+  deleteButton = 'X';
   sortButton = "Sort Button";
 
   count = 1;
   list: any = [];
   max: number = 50;
-  id = 1;
+  id: any;
```
Comment (Shitikyan) [inline src/app/create-card/create-card.component.ts]: where this used?

File: src/app/delete-button/delete-button.component.ts [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import { Component, EventEmitter, Input, Output } from '@angular/core';
+
+@Component({
+  selector: 'app-delete-button',
+  templateUrl: './delete-button.component.html',
+  styleUrls: ['./delete-button.component.css']
+})
+export class DeleteButtonComponent {
+  @Input() item: any = []
```
Comment (KarenChilingaryan) [inline src/app/delete-button/delete-button.component.ts]: it is not array

File: src/app/create-card/create-card.component.ts [hunk 4]
```diff
@@ -1,52 +1,47 @@
...
   }
-  booksByStoreID: any;
 
-  ngOnInit(): void {
+  randomNum() {
+    this.randomNumber = Math.floor(Math.random() * this.max + 1);
+  }
 
+  uniqId() {
+    this.randomId = Math.floor(Math.random() * this.max + 1)
```
Comment (KarenChilingaryan) [inline src/app/create-card/create-card.component.ts]: it isn't unique ID , you can return value and randomNumber don't declare

File: src/app/create-card/create-card.component.ts [hunk 5]
```diff
@@ -1,52 +1,47 @@
...
   randomId: any;
 
   click = () => {
     this.count++
   }
-  booksByStoreID: any;
 
-  ngOnInit(): void {
+  randomNum() {
+    this.randomNumber = Math.floor(Math.random() * this.max + 1);
```
Comment (KarenChilingaryan) [inline src/app/create-card/create-card.component.ts]: you can return value and randomNumber don't declare

**Pull request merged**